### PR TITLE
Modified tasks to run with Default Executor

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

File: framework/calico_framework.py [hunk 1]
```diff
@@ -567,46 +567,51 @@ class NotEnoughResources(Exception):
...
-    test = TestCase([sleep_task, ping_task],
-                    name="Same-Host Different-Netgroups Can't Ping")
-    scheduler.tests.append(test)
-
-    # Tasks without calico can still use original Mesos Networking
+    ping_task = PingTask(netgroups=['netgroup_a'], slave=1,
+                         can_ping_targets=[sleep_task])
+    scheduler.tests.append(TestCase([sleep_task, ping_task], name=test_name))
+
+    test_name = "Different-Host Same-Netgroups Can Ping (Default Executor)"
```
Comment (djosborne) [inline framework/calico_framework.py]: I modified the text formatting of Tests in this PR, but the important thing to note is the 2 new tests with `(Default Executor)` in their title.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM mesosphere/mesos-modules-dev-phusion:0.25.0-rc1
+FROM spikecurtis/mesos-modules-dev-phusion:taskinfo-container
```
Comment (spikecurtis) [inline Dockerfile]: Think we should merge this into the integration branch?

I guess we can change it back later once Mesos is fixed, and we'd have a way to ensure we don't have regressions.

@karya0 what do you think?

File: framework/tasks.py [hunk 3]
```diff
@@ -49,19 +50,25 @@ def as_new_mesos_task(self):
...
+            executor.executor_id.value = "execute Task %s" % self.task_id
+            executor.command.value = "python %s" % self.executor_script
+            executor.name = "Test Executor for Task %s" % self.task_id
+            executor.source = "python_test"
+            executor.container.type = mesos_pb2.ContainerInfo.MESOS
+            task.executor.MergeFrom(executor)
+            self.executor_id = executor.executor_id.value
+        else:
+            task.container.type = mesos_pb2.ContainerInfo.MESOS
+            # task.command.value = "ping google.com"
```
Comment (spikecurtis) [inline framework/tasks.py]: experimental line left commented out?

**Comment by spikecurtis** [comment]
LGTM

File: Dockerfile (hunk 2 above)
Comment (karya0) [inline Dockerfile]: Yes, let's do that.

**Pull request merged**